cmd/otelcontribcol/builder-config.yaml

```diff
@@ -6,8 +6,8 @@ dist:
   otelcol_version: 0.85.0
```

Comment:
Should we bump this version since we're upgrading core?

Reply:
Done